### DISCUSSION TITLE
What is meant by a depth of "16/8-bit"

### DISCUSSION THREAD

**Discussion by tekwiz** [ImageMagick/ImageMagick#6165] [Help] [answered]
### ImageMagick version

7.1.0-60

### Operating system, version and so on

macOS 12.6.3, installed via Homebrew

### Description

I am using identify to get information about a PDF, and it shows a depth of `16/8-bit`.  I have a fair understanding of file-format bit-depth based on [the section in the Basic Usage documentation](https://imagemagick.org/Usage/basics/#depth), but I'm unsure what "16/8-bit" is implying.  You can find the complete output in this gist: https://gist.github.com/tekwiz/7ce63ed8a784061c2ebf8af69f31ff38

Excerpt of depth & channel-depths:
```
  Depth: 16/8-bit
  Channel depth:
    Red: 8-bit
    Green: 8-bit
    Blue: 8-bit
    Alpha: 8-bit
```

Excerpt of the alpha value, color count, and beginning & end of the histogram:
```
  Alpha: srgba(255,255,255,0)   #FFFFFFFFFFFF0000
  Colors: 903
  Histogram:
        402045: (0,0,0,0) #0000000000000000 none
           474: (0,0,0,6425) #0000000000001919 srgba(0,0,0,0.0980392)
            95: (0,0,0,3341) #0000000000000D0D srgba(0,0,0,0.0509804)
            31: (0,0,0,1799) #0000000000000707 srgba(0,0,0,0.027451)
...
          1443: (65535,63479,57825,65535) #FFFFF7F7E1E1FFFF srgba(255,247,225,1)
             1: (65535,63479,57825,43690) #FFFFF7F7E1E1AAAA srgba(255,247,225,0.666667)
             1: (65535,63479,58339,13878) #FFFFF7F7E3E33636 srgba(255,247,227,0.211765)
            10: (65535,64250,61166,65535) #FFFFFAFAEEEEFFFF srgba(255,250,238,1)
         22015: (65535,65535,65535,0) #FFFFFFFFFFFF0000 srgba(255,255,255,0)
          9235: (65535,65535,65535,65535) #FFFFFFFFFFFFFFFF white
```

Excerpt of other simple colors:
```
  Matte color: grey74
  Background color: white
  Border color: srgb(223,223,223)
```

**Comment by urban-warrior** [comment, marked as answer]
16/8 bit means that the pixel container is 16 bits [0..65535] internally but the pixels are 8-bit [0..255] meaning they can be exported to an 8-bit container (e.g., BMP) without any quantization.
